### ISSUE TITLE
Cell Comments are treated like normal text content

### ISSUE THREAD

**Issue by toesus** [pyexcel/pyexcel-ods#24]
When reading a file which contains cells with comments, these comments are added to the text of the cell.
```xml
<table:table-cell table:style-name="ce227" office:value-type="string" calcext:value-type="string">
<office:annotation draw:style-name="gr2" draw:text-style-name="P2" svg:width="82.18pt" svg:height="26.62pt" svg:x="94.25pt" svg:y="0pt" draw:caption-point-x="-17.29pt" draw:caption-point-y="14.68pt"><dc:date>2017-08-25T00:00:00</dc:date>
<text:p text:style-name="P1">comment</text:p>
</office:annotation>
<text:p>key</text:p>
</table:table-cell>
```

The content of this cell after reading with pyexcel-ods will be `comment\nkey`

I think this is due to the way it is handled in `def __read_text_cell(self, cell):` [link](https://github.com/pyexcel/pyexcel-ods/blob/f5bbf7e5fd8f135b24e070e4e8e78d263cc99730/pyexcel_ods/odsr.py#L92)

It simply reads all the elements by type P, and also gets the comment-paragraph from the `<office:annotation>` element. Afterwards all paragraphs are joined together.

Would it be possible to look for P elements only as direct children of the table-cell?

**Comment by chfw** [comment]
Do mean you by ignoring <office:annotation> in <table-cell>?

**Comment by toesus** [comment]
Yes, why not. Or maybe even ignore all `text:p` which are not direct children of `table-cell`, but i'm not sure if this would be too restrictive...
So i guess for me ignoring `office:annotation` would be ok.

**Comment by vog** [comment]
Has there been any progress on this issue in the meantime?

**Comment by chfw** [comment]
thanks for your interest in pyexcel-ods. It is released in 0.5.3.
